UK/Area/London TC/TC South.prf

```diff
@@ -3,12 +3,12 @@ Settings	SettingsfilePROFILE	\..\..\Data\Settings\Local\London\TC_S_Profiles.txt
 Settings	SettingsfileSCREEN	\..\..\Data\Settings\LTC\LTC_Screen.txt
 Settings	SettingsfileVOICE	\..\..\Data\Settings\Voice.txt
 Settings	Settingsfile	\..\..\Data\Settings\LTC\LTC_General.txt
-Settings	SettingsfileARR	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileARR	\..\..\Data\Settings\Lists_SMR.txt
 Settings	SettingsfileCONFLICT	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileDEP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileFP	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSEL	\..\..\Data\Settings\Lists.txt
-Settings	SettingsfileSIL	\..\..\Data\Settings\Lists.txt
+Settings	SettingsfileDEP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileFP	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSEL	\..\..\Data\Settings\Lists_SMR.txt
+Settings	SettingsfileSIL	\..\..\Data\Settings\Lists_SMR.txt
```

Comment:
To fix

Reply:
Fixed via https://github.com/VATSIM-UK/uk-controller-pack/pull/518/commits/e6798472c371c3cddcb6745d406d63b0275d07cd